google/cloud/spanner_dbapi/cursor.py

```diff
@@ -208,9 +207,6 @@ def execute(self, sql, args=None):
             self.connection.run_prior_DDL_statements()
 
             if not self.connection.autocommit:
-                if classification == parse_utils.STMT_UPDATING:
-                    sql = parse_utils.ensure_where_clause(sql)
-
                 if classification != parse_utils.STMT_INSERT:
```

Comment:
Do we still need this? As in; do we handle an insert in any different way than for example an `UPDATE` DML statement?